cmd/kured/main.go

```diff
@@ -110,6 +115,16 @@ func main() {
 	}
 }
 
+func flagcheck(cmd *cobra.Command, args []string) {
+	if slackHookURL != "" && notifyURL != "" {
+		log.Warnf("Cannot use both --notify-url and --slack-hook-url flags. Kured will use --notify-url flag only...")
+		slackHookURL = ""
+	}
+	if slackChannel != "" || slackHookURL != "" || slackUsername != "" {
+		log.Warnf("Deprecated flag(s). Please use --notify-url flag instead.")
+	}
+}
```

Comment:
This looks good!

nit: I would also have an extra check (something we should have had in the past):
check, if `slack-hook-url` is set, that `slack-channel` is also set. Not necessary for this PR though.